tests/influxdb/helper_test.py

```diff
@@ -0,0 +1,155 @@
+        TestSeriesHelper.MySeriesHelper(server_name='us.east-1', time=159)
+        TestSeriesHelper.MySeriesHelper(server_name='us.east-1', time=158)
+        TestSeriesHelper.MySeriesHelper(server_name='us.east-1', time=157)
+        TestSeriesHelper.MySeriesHelper(server_name='us.east-1', time=156)
+        expectation = [{'points': [[159, 'us.east-1'],
+                                   [158, 'us.east-1'],
+                                   [157, 'us.east-1'],
+                                   [156, 'us.east-1']],
+                        'name': 'events.stats.us.east-1',
+                        'columns': ['time', 'server_name']}]
+
+        rcvd = TestSeriesHelper.MySeriesHelper._json_body_()
+        self.assertTrue(all([el in expectation for el in rcvd]) and
+                        all([el in rcvd for el in expectation]),
+                        'Invalid JSON body of time series returned from '
+                        '_json_body_ for one series name: {}.'.format(rcvd))
+        TestSeriesHelper.MySeriesHelper._reset_()
+        self.assertEqual(
+            TestSeriesHelper.MySeriesHelper._json_body_(),
+            [],
+            'Resetting helper did not empty datapoints.')
+
+    def testSeveralSeriesNames(self):
+        '''
+        Tests JSON conversion when there is only one series name.
+        '''
+        TestSeriesHelper.MySeriesHelper(server_name='us.east-1', time=159)
+        TestSeriesHelper.MySeriesHelper(server_name='fr.paris-10', time=158)
+        TestSeriesHelper.MySeriesHelper(server_name='lu.lux', time=157)
+        TestSeriesHelper.MySeriesHelper(server_name='uk.london', time=156)
+        expectation = [{'points': [[157, 'lu.lux']],
+                        'name': 'events.stats.lu.lux',
+                        'columns': ['time', 'server_name']},
+                       {'points': [[156, 'uk.london']],
+                        'name': 'events.stats.uk.london',
+                        'columns': ['time', 'server_name']},
+                       {'points': [[158, 'fr.paris-10']],
+                        'name': 'events.stats.fr.paris-10',
+                        'columns': ['time', 'server_name']},
+                       {'points': [[159, 'us.east-1']],
+                        'name': 'events.stats.us.east-1',
+                        'columns': ['time', 'server_name']}]
+
+        rcvd = TestSeriesHelper.MySeriesHelper._json_body_()
+        self.assertTrue(all([el in expectation for el in rcvd]) and
+                        all([el in rcvd for el in expectation]),
+                        'Invalid JSON body of time series returned from '
+                        '_json_body_ for several series names: {}.'
+                        .format(rcvd))
+        TestSeriesHelper.MySeriesHelper._reset_()
+        self.assertEqual(
+            TestSeriesHelper.MySeriesHelper._json_body_(),
+            [],
+            'Resetting helper did not empty datapoints.')
+
+    def testInvalidHelpers(self):
```

Comment:
Can this be broken down into several tests with validation of the error strings please?